### PR TITLE
Globalize using strong-globalize

### PR DESCRIPTION
**Pull request by gunjpan**
Connect to strongloop/loopback/issues/2422

### CODE REVIEW

**Comment by jannyHou** [comment]
LGTM 👍

**Comment by gunjpan** [comment]
@slnode test please

File: index.js [hunk 1]
```diff
@@ -3,4 +3,6 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-module.exports = require('./lib/jsonrpc-connector');
\ No newline at end of file
+var SG = require('strong-globalize');
+SG.SetRootDir(__dirname);
```
Comment (superkhau) [inline index.js]: nit: add a blank line to logically separate items

**Comment by superkhau** [comment]
LGTM

File: index.js (hunk 1 above)
Comment (gunjpan) [inline index.js]: done

**Comment by gunjpan** [comment]
@slnode test please

**Comment by gunjpan** [comment]
merged the commit as all CI tests were green except one which failed on `npm install` for [[cis-jenkins] PR Builder](https://cis-jenkins.swg-devops.com/job/npr/job/loopback-connector-jsonrpc~master/4/console), which was passing on a previous run.

**Pull request merged**